src/index.ts

```diff
@@ -58,6 +58,7 @@ ${chalk.bold('Options:')}
   --babel                   Transpile installed dependencies. Also enabled with "--optimize".
   --include [glob]          Auto-detect imports from file(s). Supports glob.
   --exclude [glob]          Exclude files from --include. Follows glob’s ignore pattern.
+  --config [path]           Location of Snowpack config file.
```

Comment:
I don’t know what order these should be in and at this point I’m too afraid to ask

Reply:
I feel like alphabetical order is pretty standard but I don't really mind - happy to go with the general consensus!

Reply:
I’m also big on alphabetizing! But I too will go with whatever order makes sense

Reply:
+1 to alphabetical